Samples/1c-multiple-input-swaggers/Client/IStorageManagementClient.cs

```diff
@@ -0,0 +1,63 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
```

Comment:
hmmm seems like some PR in the past overruled CI in the past when it really shouldn't, because this kind of change is unacceptable :-( will investigate, but it's fine to submit those with this PR